src/module.ts

```diff
@@ -158,6 +158,9 @@ export default defineNuxtModule<ModuleOptions>({
     const resolvedConfig = resolveConfig(tailwindConfig)
     // Avoid creating null plugins for intelisense
     resolvedConfig.plugins = []
+    resolvedConfig.presets = resolvedConfig.presets.map(
+      preset => preset.plugins != null ? { ...preset, plugins: [] } : preset
```

Comment:
Would this not be?

```suggestion
      preset => preset.plugins !== null ? { ...preset, plugins: [] } : preset
```

Reply:
Due to the way [plugins are resolved by Tailwind](https://github.com/tailwindlabs/tailwindcss/blob/master/src/util/resolveConfig.js#L219) from a preset or base config, `{}`, `{ plugins: null }`, and `{ plugins: undefined }` are all cases we should leave alone. 

`preset.plugins != null` is equivalent to `preset.plugins !== null && preset.plugins !== undefined`. I suppose we could put the more verbose version in?

Reply:
I'm no expert - just wondered about the reasoning for this! It would be the maintainers of the module providing their advice, but based on your answer, what about `{ plugins: "" }` (empty string) - such cases are falsy right? As far as I know, using `!==` is highly preferable otherwise just using `preset.plugins ? { ...preset } : preset` (or wrap in `Boolean()`) could cover all falsy conditions.